File: namsor/python/src/GenderJapaneseNameFullExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.JapaneseApi(api_client).gender_japanese_name_full(
            japanese_name="中松 義郎",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling JapaneseApi#gender_japanese_name_full: %s\n" % e)

File: namsor/ruby/src/GenderJapaneseNameFullExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::JapaneseApi.new.gender_japanese_name_full(
        "中松 義郎", # japanese_name
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling JapaneseApi#gender_japanese_name_full: #{e}"
end
